### DISCUSSION TITLE
Tuning microtonal notes

### DISCUSSION THREAD

**Discussion by ghost** [mido/mido#414] [Q&A]
Is it possible to do a microtonal tuning on a single midi-note of a specific track? And if so how?

**Comment by ruckyboy** [comment]
I've been looking into micro-tonal and midi.
From what I can make out in the midi specs, it's not supported in any real way.
Using the pitch-wheel you can adjust the tuning - very finely - but that effects the whole channel, not just a single note. Midi spec suggests the range is  ±2 semitones, though devices and software can vary upto  ±12 semitones in some cases.
Depending on your requirements though, you could try setting up a specific channel that just plays that single micro-tonal note using pitch-wheel?

Just off the top of my head.
